idp-service/src/main/resources/application-local.properties

```diff
@@ -5,8 +5,11 @@
 ## ------------------------------------------------- IdP ---------------------------------------------------------------
 mosip.idp.misp.license.key=idp-dev-misp-lic-key
 mosip.idp.amr-acr-mapping-file-url=https://raw.githubusercontent.com/mosip/mosip-config/develop-v3/amr-acr-mapping.json
-mosip.idp.post-req-client-mgmt-scope=SCOPE_add_oidc_client
-mosip.idp.put-req-client-mgmt-scope=SCOPE_update_oidc_client
+
+mosip.idp.clientmgmt.postcreateclient=SCOPE_add_oidc_client
+mosip.idp.clientmgmt.putupdateclient=SCOPE_update_oidc_client
```

Comment:
should we have post and put in the property name, can we rather call add and update so it is less REST technology oriented and more functionality oriented.

Reply:
Changed the property name to mosip.idp.clientmgmt.create-client  and mosip.idp.clientmgmt.update-client